raven/config/commands.py

```diff
@@ -0,0 +1,86 @@
+"""
+Author(s):      Carson Schubert (carson.schubert14@gmail.com)  
+Date Created:   04/05/2019
+
+Command group for setting raven configuration.
+"""
+
+import click
+import yaml
+from pathlib import Path
+from colorama import init, Fore
+from raven.utils.question import user_input, user_confirms
+from raven.utils.config import CONFIG_FIELDS
+from raven.utils.local_cache import global_cache
+from raven.utils.config import get_config, update_config
+
+init()
+    
+### OPTIONS ###
+
+
+### COMMANDS ###
+@click.group()
+@click.pass_context
+def config(ctx):
```

Comment:
First line should be summary, second line should be a detailed explanation along with args and what it returns

Reply:
https://github.com/google/styleguide/blob/gh-pages/pyguide.md